File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.java
// Last updated: 18/10/2025, 09:01:22
class Solution {
    public boolean checkIfPangram(String s) {
       	   boolean attendance[] = new boolean[26];

        for (char ch : s.toLowerCase().toCharArray()) {
            if (ch >= 'a' && ch <= 'z') {
                attendance[ch - 'a'] = true;
            }
        }

        for (boolean present : attendance) {
            if (!present) return false;
        }

        return true;
    }
}

File: 1960-CheckIfTheSentenceIsPangram/1960-CheckIfTheSentenceIsPangram.cpp
// Last updated: 18/10/2025, 09:01:17
class Solution {
public:
    bool checkIfPangram(string sentence) {
        vector<bool>alpha(26,0);
        for(int i = 0;i<sentence.size();i++)
        {
            alpha[sentence[i]-'a'] = 1;
        }

        for(int i = 0;i<26;i++)
        {
            if(alpha[i] == 0)
            return 0;
        }

        return 1;
    }
};